### PR TITLE
packages updated

### PR DESCRIPTION
**Pull request by TrejGun**
Hi there
i have updated packages to latest versions

the only thing i did - removed air plus
i like it but there is absolutely no need to use eslint-react-plugin
so it can be done this way
airplus can be splitted into `reat` and `base` where react extends base
if you like i can do a PR for this too

### CODE REVIEW

File: src/__tests__/commonTest.js [hunk 1]
```diff
@@ -35,23 +35,24 @@ describe('commonMerger', () => {
             hostname: 'testhost',
             tagSeparator: '#'
         })
-        .then(loadedConfig => {
-            assert.deepEqual(loadedConfig, testConfig)
-        })
+            .then(loadedConfig => {
+                assert.deepEqual(loadedConfig, testConfig)
+                done()
```
Comment (zerkalica) [inline src/__tests__/commonTest.js]: Yo don't need done, just return promise https://mochajs.org/#asynchronous-code: Working with Promises

File: package.json [hunk 2]
```diff
@@ -13,24 +13,19 @@
     "release-patch": "npm run pull && npm version patch && npm run push",
     "release-minor": "npm run pull && npm version minor && npm run push",
     "release-major": "npm run pull && npm version major && npm run push",
-    "clean": "rm -rf *.map loadConfig.js webpack.js index.js common filters parsers utils __tests__",
-    "build": "npm run clean && babel src --source-maps --out-dir .",
+    "clean": "rm -rf ./build",
+    "build": "npm run clean && babel -d ./build ./src -s",
```
Comment (zerkalica) [inline package.json]: How to use node-config-loader/webpack entry point ?

File: src/__tests__/commonTest.js (hunk 1 above)
Comment (TrejGun) [inline src/__tests__/commonTest.js]: you are right

**Comment by TrejGun** [comment]
fixed, please merge

**Comment by TrejGun** [comment]
Hi Stefan, any problem to merge my PR?

**Comment by zerkalica** [comment]
Some problems with applications, which use this lib and babel5. I bump to 2.0.0 version.

**Comment by zerkalica** [comment]
published as 2.0.0

**Comment by TrejGun** [comment]
thanks

**Pull request merged**